sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -500,7 +497,6 @@ main();
 
 ## Known Issues
 
-- `beginAnalyzeHealthcareEntities` is still in gated preview and can not be used with AAD credentials. For more information, see [the Text Analytics for Health documentation](https://docs.microsoft.com/azure/cognitive-services/text-analytics/how-tos/text-analytics-for-health?tabs=ner#request-access-to-the-public-preview).
 - At time of this SDK release, the `modelVersion` option to `beginAnalyzeHealthcareEntities` is ignored by the service. The service always processes the operation using the "latest" model.
```

Comment:
Not related to this PR, but if we can clean this line too, that would be great.
After GA, the service will start adding model-version. See comment here: https://msazure.visualstudio.com/Cognitive%20Services/_workitems/edit/9479280

Also, after looking at the [docs](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/concepts/model-versioning#available-versions), and passing `model-version=2021-05-15`, the service didn't fail or anything

Reply:
Done :)